include/gsl/multi_span

```diff
@@ -1517,85 +1559,86 @@ public:
         return Ret{data_ + ridx, bounds_.slice()};
     }
 
-    constexpr iterator begin() const GSL_NOEXCEPT { return iterator{this, true}; }
+    constexpr iterator begin() const noexcept { return iterator{this, true}; }
 
-    constexpr iterator end() const GSL_NOEXCEPT { return iterator{this, false}; }
+    constexpr iterator end() const noexcept { return iterator{this, false}; }
 
-    constexpr const_iterator cbegin() const GSL_NOEXCEPT
+    GSL_SUPPRESS(type.1) // NO-FORMAT: attribute
+    constexpr const_iterator cbegin() const noexcept
     {
         return const_iterator{reinterpret_cast<const const_span*>(this), true};
     }
 
-    constexpr const_iterator cend() const GSL_NOEXCEPT
+    constexpr const_iterator cend() const noexcept
     {
         return const_iterator{reinterpret_cast<const const_span*>(this), false};
     }
 
-    constexpr reverse_iterator rbegin() const GSL_NOEXCEPT { return reverse_iterator{end()}; }
+    constexpr reverse_iterator rbegin() const noexcept { return reverse_iterator{end()}; }
 
-    constexpr reverse_iterator rend() const GSL_NOEXCEPT { return reverse_iterator{begin()}; }
+    constexpr reverse_iterator rend() const noexcept { return reverse_iterator{begin()}; }
 
-    constexpr const_reverse_iterator crbegin() const GSL_NOEXCEPT
+    constexpr const_reverse_iterator crbegin() const noexcept
     {
         return const_reverse_iterator{cend()};
     }
 
-    constexpr const_reverse_iterator crend() const GSL_NOEXCEPT
+    constexpr const_reverse_iterator crend() const noexcept
     {
         return const_reverse_iterator{cbegin()};
     }
 
     template <typename OtherValueType, std::ptrdiff_t... OtherDimensions,
-              typename = std::enable_if_t<std::is_same<
-                  std::remove_cv_t<value_type>, std::remove_cv_t<OtherValueType>>::value>>
-    constexpr bool
-    operator==(const multi_span<OtherValueType, OtherDimensions...>& other) const GSL_NOEXCEPT
+              typename = std::enable_if_t<std::is_same<std::remove_cv_t<value_type>,
+                                                       std::remove_cv_t<OtherValueType>>::value>>
+    constexpr bool operator==(const multi_span<OtherValueType, OtherDimensions...>& other) const
+        noexcept
```

Comment:
Previously the `GSL_NOEXCEPT` was on the same line. If there is no reason to put it on a new line, I would leave it there. Or is this something the editor automatically does?

Reply:
yeah, this is clang-format